test/test_pipeline_baseline.py

```diff
@@ -85,14 +86,16 @@ def _test_error(pipeline_name, language, config, pipeline_kwargs,
         execute_pipeline.main(args)
 
 
-@mock.patch('pipeline.utils.task_utils.run_gradle_task')
+@mock.patch('pipeline.utils.task_utils.get_gradle_task_output')
+@mock.patch('pipeline.tasks.protoc_tasks.PythonChangePackageTask._copy_proto')
```

Comment:
Can we just mock the entire `PythonChangePackageTask` (or its `execute`)? That would simplify the setup of the whole test -- you don't need several yaml files in the testcase (and I assume that this task is well tested in another test file).